app/src/components/TransactionStatus.vue

```diff
@@ -50,7 +50,7 @@
 <script lang="ts">
 import { computed, defineComponent, Ref, ref, SetupContext, watch } from 'vue';
 import { getEtherscanUrl } from 'src/utils/utils';
-import useWalletStore from 'src/store/wallet';
+import { DEFAULT_PROVIDER } from 'src/utils/chains';
```

Comment:
I think I've found a few other instances we use the user's provider to read contract data, that we need to change to `DEFAULT_PROVIDER`. These link to lines on the last commit in this branch:

- L543 `cart.ts` :https://github.com/dcgtc/dgrants/blob/cc8f5485f8035b3a73e0eec34bea4ed351932734/app/src/store/cart.ts#L543
- L549 `cart.ts` :https://github.com/dcgtc/dgrants/blob/cc8f5485f8035b3a73e0eec34bea4ed351932734/app/src/store/cart.ts#L549
- L188 and L189 of `app/src/utils/utils`: https://github.com/dcgtc/dgrants/blob/cc8f5485f8035b3a73e0eec34bea4ed351932734/app/src/utils/utils.ts#L188-L189
- L464 of `grantRound.ts`: https://github.com/dcgtc/dgrants/blob/cc8f5485f8035b3a73e0eec34bea4ed351932734/app/src/utils/data/grantRounds.ts#L464
- L534 and 536 of `grantRound.ts`: https://github.com/dcgtc/dgrants/blob/cc8f5485f8035b3a73e0eec34bea4ed351932734/app/src/utils/data/grantRounds.ts#L534-L536